### ISSUE TITLE
empty canvas after switching to new vscode extension

### ISSUE THREAD

**Issue by ptaranto** [statelyai/xstate-vscode#39]
no machine is rendered in the new visualizer tab either when using:

- `createMachine({...})`
```
const promiseMachine = createMachine({
  id: 'promise',
  initial: 'pending',
  states: {
    pending: {
      on: {
        // state transition (shorthand)
        // this is equivalent to { target: 'resolved' }
        RESOLVE: 'resolved',

        // state transition (object)
        REJECT: {
          target: 'rejected',
        },
      },
    },
    resolved: {
      type: 'final',
    },
    rejected: {
      type: 'final',
    },
  },
})
```
 or
- `model = createModel(); model.createMachine({...})`
```
import { createModel } from 'xstate/lib/model'
import { createMachine } from 'xstate'

interface Config {
  featureIsEnabled?: boolean
}

const userFlowModel = createModel(
  {
    config: {} as Config,
  },
  {
    events: {
      TEST_SELECTED: () => ({}),
    },
  }
)

export const userFlowMachine = userFlowModel.createMachine(
  {
    id: 'user-flow',
    initial: 'selectTest',
    states: {
      selectTest: {
        on: {
          TEST_SELECTED: 'test',
        },
      },
      test: {},
      results: {},
    },
  },
  {
    actions: {},
  }
)
```

**Comment by ptaranto** [comment]
PS: old extension was only disabled tho, does it have to be fully uninstalled?

**Comment by mattpocock** [comment]
@ptaranto Maybe try fully uninstalling it?

**Comment by mattpocock** [comment]
Fixed in 1.5.3

**Comment by wKovacs64** [comment]
> Fixed in 1.5.3

Still blank for me on 1.5.4 (Windows, if that matters - but no errors in VSCode "Window" log).
